Sources/SwiftKueryPostgreSQL/PostgreSQLConnection.swift

```diff
@@ -484,6 +511,7 @@ public class PostgreSQLConnection: Connection {
             }
 
             PQclear(result)
+            setState(.idle)
```

Comment:
Same comment as above - could move single `setState(.idle)` to above the `if status`